tests/Mutator/Unwrap/UnwrapArrayUdiffUassocTest.php

```diff
@@ -132,7 +132,7 @@ public function provideMutationCases(): \Generator
             <<<'PHP'
 <?php
 
-$a = array_udiff_uassoc($foo->bar(), $foo->baz());
+$a = array_udiff_uassoc($foo->bar(), $foo->baz(), $callback);
```

Comment:
this one [has two](http://php.net/manual/ru/function.array-udiff-uassoc.php) callbacks

Reply:
Thank you, @borNfreee!